### PR TITLE
Ignoring private info

### PR DESCRIPTION
**Pull request by jameslesliecarter**
Need to keep private keys private as well as reducing cruft by not pushing node modules or bundled code.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+node_modules
```
Comment (ghost) [inline .gitignore:1]: big folder

**Review by jnalls13 — APPROVED** [review]
Looks good guy!

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+public/bundle.js
```
Comment (ghost) [inline .gitignore:2]: this is filled with gobbledygook

File: .gitignore [hunk 3]
```diff
@@ -0,0 +1,3 @@
...
+config.js
```
Comment (ghost) [inline .gitignore:3]: private info

**Review by ghost — APPROVED** [review]
b

**Pull request merged**